fluent-syntax/src/stream.js

```diff
@@ -24,6 +26,7 @@ export class ParserStream {
     }
 
     this.index++;
+    this.checkLineBreak();
```

Comment:
I think it's very risky. We're not guaranteed to always advance the stream with next, and we may want to have methods that allow us to skip chars in the future (like "skipToX").

I would much prefer to calculate the line at the time we encounter an error.

Since the error scenario is already an exception, I believe the performance doesn't matter, and just going through the source and counting number of '\n' in it should be enough).